openshift_day2_configurator/configurators/ldap.py

```diff
@@ -440,66 +440,50 @@ def execute_ldap_configuration(
     config: Dict[str, Any],
     logger: logging.Logger,
     client: DynamicClient,
-    progress: Progress | None = None,
-) -> Dict[str, Dict[str, str | bool]]:
+    progress: Optional[Progress] = None,
+) -> Dict[str, Dict[str, Union[str, bool]]]:
```

Comment:
why is this change added ? original method id preferred (see https://docs.python.org/3/library/typing.html#typing.Union)

Reply:
same applies for `Optional`

Reply:
done